src/Shared/TagInfo_Kaart.xml

```diff
@@ -469,6 +469,7 @@
 	<tag key="highway" value="proposed" name="Proposed" type="way"/>
 	<tag key="highway" value="raceway" name="Raceway" type="way"/>
 	<tag key="highway" value="residential" name="Residential" type="way" description="Road in a residential area" lineColor="FFFFFF" lineWidth="2.000000"/>
+ 	<tag key="highway" value="residential" name="Motorcycle only" type="way" belongsTo="motorcycle=yes;motor_vehicle=no" description="Road in a residential area where only motorcycles are allowed" lineColor="0000FF" lineWidth="2.000000" /> 
```

Comment:
In the other PR you had `lineColor="FFFFFF"` and `lineWidth="1.000000"`. I would pick whichever matches the JOSM paint style better.